pkg/cri/remote_runtime.go

```diff
@@ -0,0 +1,95 @@
+package cri
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"net"
+	"strings"
+	"time"
+
+	"github.com/golang/glog"
+	"google.golang.org/grpc"
+
+	runtimeapi "k8s.io/cri-api/pkg/apis/runtime/v1alpha2"
+)
+
+const (
+	DefaultConnectionTimeout = 15 * time.Second
+	maxMsgSize               = 1024 * 1024 * 16 // 16 MB
+)
+
+// remoteRuntimeService is a gRPC implementation of RuntimeService.
+type remoteRuntimeService struct {
+	timeout       time.Duration
+	runtimeClient runtimeapi.RuntimeServiceClient
```

Comment:
Just double-checking here, I assume that there is now `close()` method or other appropriate clean-up method that we need to expose via interface for this guy?

Reply:
`RuntimeServiceClient` itself doesn't provide such a method, however, I think you're right, so I implemented `Close()` method for `RuntimeService` interface that cleanup stale gRPC connections.